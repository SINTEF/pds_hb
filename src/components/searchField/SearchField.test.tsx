import React from 'react'
import renderer from 'react-test-renderer'
import { SearchField } from '.'

it('renders primary without crashing', () => {
  const tree = renderer
    .create(
      <SearchField
        onValueChanged={() => {
          return
        }}
        defaultValue=""
        icon="search"
        suggestions={['test', 'test2']}
        variant="primary"
      />
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders standard without crashing', () => {
  const tree = renderer
    .create(
      <SearchField
        onValueChanged={() => {
          return
        }}
        defaultValue=""
        icon="search"
        suggestions={['test', 'test2']}
        variant="primary"
      />
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
