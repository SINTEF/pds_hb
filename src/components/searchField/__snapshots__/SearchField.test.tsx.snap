// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders primary without crashing 1`] = `
<div
  className="SearchField"
>
  <div
    className="primary field"
  >
    <input
      onChange={[Function]}
      onKeyDown={[Function]}
    />
    <i
      className="material-icons icon"
    >
      search
    </i>
  </div>
  <ul
    className="filtered"
  />
</div>
`;

exports[`renders standard without crashing 1`] = `
<div
  className="SearchField"
>
  <div
    className="primary field"
  >
    <input
      onChange={[Function]}
      onKeyDown={[Function]}
    />
    <i
      className="material-icons icon"
    >
      search
    </i>
  </div>
  <ul
    className="filtered"
  />
</div>
`;
